websocket_handler.c

```diff
@@ -66,7 +71,6 @@ esp_websocket_client_handle_t websocket_network_manager()
  */
 int websocket_send_data(esp_websocket_client_handle_t network_handle, char* payload)
```

Comment:
Can you fix whitespaces in this function

Reply:
whitespace as in tabs and space?

Reply:
Yes